qrisk/stats.py

```diff
@@ -314,10 +323,18 @@ def omega_ratio(returns, annual_return_threshhold=0.0):
 
     """
 
-    daily_return_thresh = pow(1 + annual_return_threshhold, 1 /
-                              APPROX_BDAYS_PER_YEAR) - 1
+    if returns.size < 2:
```

Comment:
`len` is the standard way to talk about the length of some collection, `size` has an overloaded meaning in numpy/pandas, for example:

```
In [52]: len(np.arange(25).reshape(5, 5))
Out[52]: 5

In [53]: np.arange(25).reshape(5, 5).size
Out[53]: 25
```

I think it is just more standard